api/src/controllers/educonnect.js

```diff
@@ -0,0 +1,59 @@
+const express = require("express");
+const router = express.Router();
+const passport = require("passport");
+const saml2 = require("saml2-js");
+const fs = require("fs");
+const { capture } = require("../sentry");
+const path = require("path");
+const fetch = require("node-fetch");
+const { ERRORS } = require("../utils");
+
+const { EDUCONNECT_SP_CERT, EDUCONNECT_SP_KEY, EDUCONNECT_IDP_CERT } = require("../config.js");
+
+let sp_options = {
+  entity_id: "SNU",
+  certificate: EDUCONNECT_SP_CERT,
+  private_key: EDUCONNECT_SP_KEY,
+  assert_endpoint: "https://app-a29a266c-556d-4f95-bc0e-9583a27f3f85.cleverapps.io/educonnect/callback",
+};
+let sp = new saml2.ServiceProvider(sp_options);
+
+// Create identity provider
+var idp_options = {
```

Comment:
```suggestion
const idp_options = {
```

On évite les `var`, mais c'est pas hyper important. Aussi on fait plutôt des variables/constantes en `lowerCamelCase` vs `snake_case`, mais c'est ignorable aussi.